docs/contributing/unit_tests.rst

```diff
@@ -7,3 +7,6 @@ New code is highly encouraged to have basic unit tests that demonstrate its func
 Request that has failing unit tests cannot be merged.
 
 The unit tests for both `pulpcore` and `pulpcore-plugin` live in the tests_ folder.
```

Comment:
s/folder/directory

There is no longer a top-level ```tests/``` directory, so this should clarify that the test code lives in ```pulpcore/tests/``` and ```plugin/tests/```.

The ```tests``` link on line 1 is broken for the same reason.

Reply:
This file was replaced (includes your comment) by https://github.com/pulp/pulp/pull/3435/